### PR TITLE
testing/sndio: update to 1.5.0

### PR DESCRIPTION
**Pull request by maxice8**
- Remove empty variables
- Remove superfluous 'cd's
- add $pkgname-openrc subpackage

### CODE REVIEW

File: testing/sndio/sndio.initd [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+command=/usr/bin/sndiod
+command_args="$sndio_args"
+name="sndio sound daemon"
+
+depends() {
+	need localmount
+	use net
+}
+
+start() {
```
Comment (Ikke) [inline testing/sndio/sndio.initd]: Is there a reason you need to define custom start/stop functions?

The openrc [styleguide][] advises to not define your own.

[styleguide]:https://github.com/OpenRC/openrc/blob/master/service-script-guide.md#dont-write-your-own-startstop-functions

Comment (maxice8) [reply testing/sndio/sndio.initd]: nope i just copy pasted a sndio from somewhere

File: testing/alsa-sndio/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+pkgver=0.2
+pkgrel=0
+pkgdesc="ALSA PCM to play audio on sndio servers"
+options="!check" # Has no testsuite
+url="https://github.com/Duncaen/alsa-sndio"
+arch="all"
+license="ISC"
+makedepends="alsa-lib-dev sndio-dev"
+source="$pkgname-$pkgver.tar.gz::https://github.com/Duncaen/alsa-sndio/archive/${pkgver}.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
```
Comment (TBK) [inline testing/alsa-sndio/APKBUILD]:
```suggestion
builddir="$srcdir/$pkgname-$pkgver"
```

File: testing/aucatctl/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+pkgver=0.1
+pkgrel=0
+pkgdesc="Control aucat and/or sndiod voluem through MIDI"
+url="http://www.sndio.org"
+arch="all"
+license="ISC"
+makedepends="sndio-dev"
+subpackages="$pkgname-doc"
+source="http://www.sndio.org/${pkgname}-${pkgver}.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
```
Comment (TBK) [inline testing/aucatctl/APKBUILD]:
```suggestion

```

Comment (TBK) [reply testing/aucatctl/APKBUILD]: Any tests?

File: testing/sndio/APKBUILD [hunk 4]
```diff
@@ -1,27 +1,31 @@
...
 makedepends="alsa-lib-dev"
-install=""
-subpackages="$pkgname-dev $pkgname-doc $pkgname-libs"
-source="http://www.sndio.org/sndio-$pkgver.tar.gz"
+subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-openrc"
+install="$pkgname.pre-install"
+source="http://www.sndio.org/sndio-$pkgver.tar.gz
+	sndio.initd
+	sndio.confd"
 builddir="$srcdir/sndio-$pkgver"
```
Comment (TBK) [inline testing/sndio/APKBUILD]:
```suggestion

```

Comment (TBK) [reply testing/sndio/APKBUILD]: Any tests?

File: testing/sndio/APKBUILD [hunk 5]
```diff
@@ -1,27 +1,31 @@
+# Contributor: Leo <thinkabit.ukim@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=sndio
-pkgver=1.4.0
+pkgver=1.5.0
 pkgrel=0
 pkgdesc="Small audio and MIDI framework part of the OpenBSD project"
 url="http://www.sndio.org/"
 arch="all"
 license="ISC"
```
Comment (TBK) [inline testing/sndio/APKBUILD]:
```suggestion
license="ISC"
pkgusers="sndiod"
pkggroups="sndiod"
```

File: testing/sndio/APKBUILD (hunk 4 above)
Comment (maxice8) [reply testing/sndio/APKBUILD]: nope

**Comment by maxice8** [comment]
fixed

File: testing/sndio/sndio.initd (hunk 1 above)
Comment (Ikke) [reply testing/sndio/sndio.initd]: This service file doesn't even seem to work. It looks like sndiod does not want to run as root ("sndiod: setpriority: Permission denied"). If I run it as a normal user, it just seems to work.

**Comment by maxice8** [comment]
On Sun, May 26, 2019 at 01:35:22PM -0700, Kevin Daudt wrote:
> Ikke commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,27 @@
> +#!/sbin/openrc-run
> +# Copyright 1999-2018 Gentoo Foundation
> +# Distributed under the terms of the GNU General Public License v2
> +
> +command=/usr/bin/sndiod
> +command_args="$sndio_args"
> +name="sndio sound daemon"
> +
> +depends() {
> +	need localmount
> +	use net
> +}
> +
> +start() {
> 
> This service file doesn't even seem to work. It looks like sndiod does not want to run as root ("sndiod: setpriority: Permission denied"). If I run it as a normal user, it just seems to work.
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/7256#discussion_r287613535

Worked for me at the time, i'll close this since i use pulseaudio now

**Pull request closed without merging**